app/client/src/features/accounts/ResetPasswordRequest.tsx

```diff
@@ -0,0 +1,13 @@
+import { graphql } from 'react-relay';
+
+const RESET_PASSWORD_FORM_MUTATION = graphql`
+    mutation ResetPasswordRequestMutation($input: ResetPasswordForm!) {
+        resetPassword(input: $input) {
+            isError
+            message
+            body
+        }
+    }
+`;
+
+export { RESET_PASSWORD_FORM_MUTATION }
```

Comment:
This isn't consistently named with the previous ForgotPassRequest and ForgotPassForm (Request had the jsx component and Form had the graphql mutation, but this is the opposite).  However, I have the same comment as before, please merge the files.